src/Uno.UI.Runtime.Skia.Gtk/UI/Controls/GtkTextBoxView.cs

```diff
@@ -90,6 +92,27 @@ public virtual void UpdateProperties(TextBox textBox)
 
 	public void SetFocus() => InputWidget.HasFocus = true;
 
+	public void SetClip(int clipLeft, int clipRight, int clipTop, int clipBottom)
+	{
+		if (_position.x != -1)
```

Comment:
I think it could be possible to have a "valid" position with x being -1 (i.e, one pixel of the TextBox is out of screen bounds).

I would define `_position` as `private (int x,int y)? _position` so that we know that `null` means not set.